### DISCUSSION TITLE
Does flushing certs or clearing credentials drop existing connections/SAs?

### DISCUSSION THREAD

**Discussion by Noah-Wagner** [strongswan/strongswan#365] [Q&A] [answered]
Hi! If I use vici to run "[flush-certs](https://github.com/strongswan/strongswan/blob/master/src/libcharon/plugins/vici/README.md#flush-certs)" or "[clear-creds](https://github.com/strongswan/strongswan/blob/master/src/libcharon/plugins/vici/README.md#clear-creds)" will that remove already established SAs and cause existing connections to terminate? Thanks!

**Comment by Thermi** [comment, marked as answer]
No.
